conventions/code-reviews.md

```diff
@@ -0,0 +1,8 @@
+# Code Reviews
+
+1. The implementer is responsible to drive the feature, and so to ask other developers for reviews
+2. The Second CR must be done by someone outside the original product team
+3. The Second CR should be less strict than the first one
+4. Priority is to fix forward rather than block
```

Comment:
> Priority is to fix forward rather than block. In cases where non-breaking issues are surfaced JIRA tickets must be created and referenced within the PR